Update

gemansipator opened the pull request.

gemansipator committed:
Task01 Введи с клавиатуры 5 слов в список строк. Удали 3 - ий элемент списка, и выведи оставшиеся элементы в обратном порядке.

gemansipator committed:
Task02 1. Создай список из слов "мама", "мыла", "раму".
        2. После каждого слова вставь в список строку, содержащую слово "именно".
        3. Вывести результат на экран, каждый элемент списка с новой строки.*/

gemansipator committed:
Task03 1. Создай список слов, заполни его самостоятельно.
       2. Метод fix должен:
       2.1. удалять из списка строк все слова, содержащие букву "р"
       2.2. удваивать все слова содержащие букву "л".
       2.3. если слово содержит и букву "р" и букву "л", то оставить это слово без изменений.
       2.4. с другими словами ничего не делать.

gemansipator committed:
Task04  1. Введи с клавиатуры 10 слов в список строк.
        2. Метод doubleValues должен удваивать слова по принципу:
        "альфа", "бета", "гамма" -> "альфа", "альфа", "бета", "бета", "гамма", "гамма"
        3. Выведи результат на экран, каждое значение с новой строки

gemansipator committed:
Task05  1. Введи с клавиатуры 10 слов в список строк.
         2. Определить, является ли список упорядоченным по возрастанию длины строки.
         3. В случае отрицательного ответа вывести на экран индекс первого элемента, нарушающего такую упорядоченность.

RinatBeybutov commented on src/main/java/site/javadev/lesson_13/Task02.java:
```diff
@@ -0,0 +1,19 @@
+        list.add("мама");
+        list.add("мыла");
+        list.add("раму");
+        System.out.println(list);
```
нигде не добавляется слово "именно"

RinatBeybutov commented on src/main/java/site/javadev/lesson_13/Task05.java:
```diff
@@ -0,0 +1,39 @@
+public class Task05 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        ArrayList<String> listOfLines = new ArrayList<>();
```
лучше избегать упоминания типа данных в названии переменной, лучше просто lines или strings

RinatBeybutov commented on src/main/java/site/javadev/lesson_13/Task05.java:
```diff
@@ -0,0 +1,39 @@
+
+        // Проверяем упорядоченность списка по длине строк
+        for (int i = 0; i < listOfLines.size() - 1; i++) {
+            if (listOfLines.get(i).length() > listOfLines.get(i + 1).length()) {
```
если i-Ый больше, чем i + 1, значит это убывание, а не возрастание

gemansipator committed:
lines - переименовал

gemansipator committed:
Добавлено слово "именно"
for (int i = 0; i < list.size(); i++) {
            result.add(list.get(i));
            result.add("именно");

        }
